web/src/hooks/useApps.ts

```diff
@@ -149,7 +149,7 @@ const useApps = () => {
         return;
       }
 
-      updateApp({
+      return updateApp({
```

Comment:
@michaelfurutin It will be better to update fetcher instead. Let's update this so that it can be used as
```typescript
const {trigger: toggleAppActivity} = useSWRMutation("app", updateAppFetcher, {...})
toggleAppActivity(true | false)

const {trigger: setAppName} = useSWRMutation("app", updateAppFetcher, {...})
setAppName(<name>: string)

```